modules/ztp-configuring-pgt-image-registry.adoc

```diff
@@ -0,0 +1,103 @@
+
+.Prerequisites
+
+* You have installed and configured Zero Touch Provisioning (ZTP). For information about this, see the topic on ZTP in Additional resources.
+
+.Procedure
+
+. Add the following YAML to the `StorageClass.yaml` file to configure the image registry:
++
+[source,yaml]
+----
+sourceFiles:
+   # storage class
+   - fileName: StorageClass.yaml
+     policyName: "sc-for-image-registry"
+     metadata:
+       name: image-registry-sc
+       annotations:
+         ran.openshift.io/ztp-deploy-wave: "100" <1>
+  # persistent volume claim
+   - fileName: StoragePVC.yaml
+    policyName: "pvc-for-image-registry"
+    metadata:
+      name: image-registry-pvc
+      namespace: openshift-image-registry
+        annotations:
+          ran.openshift.io/ztp-deploy-wave: "100"  <2>
+    spec:
+      accessModes:
+        ReadWriteMany
+      resources:
+        requests:
+          storage: 100Gi
+      storageClassName: image-registry-sc
+      volumeMode: Filesystem
+  # persistent volume
+  - fileName: ImageRegistryPV.yaml <3>
+    policyName: "pv-for-image-registry"
+    metadata:
+      annotations:
+        ran.openshift.io/ztp-deploy-wave: "100"  <4>
+  # image registry config
+  - fileName: ImageRegistryConfig.yaml <5>
+    policyName: "config-for-image-registry"
+    complianceType: musthave <5>
+    metadata:
+      annotations:
+        ran.openshift.io/ztp-deploy-wave: "100" <6>
+    spec:
+      storage:
+        pvc:
+          claim: "image-registry-pvc"
+----
+<1> Remove this when moved to the correct PGT (`site/group/common`).
+<2> Remove this when moved to the correct PGT (`site/group/common`).
+<3> This assumes that `mount_point` is set to `/var/imageregistry` in `SiteConfig` using StorageClass `image-registry-sc` (see the topic on configuring disk partitioning with `SiteConfig`).
+<4> Remove this when moved to the right PGT (`site/group/common`).
+<5> Configure registry to point to the PVC created above.
+<6> Remove this when moved to the right PGT (`site/group/common`)
+
+.Verification
+
+. Check that the CRD `Config` of the group `imageregistry.operator.openshift.io` instance is not reporting any errors.
+
+. A few minutes after the installation process is complete, you should see the `PersistentVolumeClaim` filling up.
+
+. Check that the `registry*` pod is up correctly located under the `openshift-image-registry` namespace.
+
+From inside the node:
+
+. Verify successful login to the registry with `podman`:
++
+[source,yaml]
+----
+  oc login -u kubeadmin -p <password_from_install_log> https://api-int.<cluster_name>.<base_domain>:6443
+  podman login -u kubeadmin -p $(oc whoami -t) image-registry.openshift-image-registry.svc:5000
+----
+
+. Check for disk partitioning using `lsblk` to list your blocks:
++
+[source,yaml]
+----
+[core@mysno ~]$ lsblk
```

Comment:
`core@mysno` suggests you have SSHed into a cluster, which is fine for development purposes, but not ideal for customer facing docs, since SSH is not recommended for production cluster nodes. Suggest using `oc/debug node` as outlined here: https://github.com/openshift/openshift-docs/pull/53083/files#diff-a8c623d5afaf11e6c3cbbd690e508160a70fdeb8cd149a0d92bccb067206efd2R129

Reply:
Resolved.